### DISCUSSION TITLE
how can I change my desired shape of pieces puzzle?

### DISCUSSION THREAD

**Discussion by benyaminrez** [flbulgarelli/headbreaker#76] [General]
Hello,
I need help
,how we can change desired shape of pieces puzzle?
I want change the pieces puzzle not rounded and not normal, i need more curve 
please help me
thanks alot

**Comment by meezyart** [comment]
I have a preset outline in svg would I be able to use that

**Reply by flbulgarelli** [reply]
:wave: Hi! Unfortunately, using an SVG file as pieces outline is not currently supported.  You'll need to define your own `outline` taking your SVG as inspiration. 

Please see https://github.com/flbulgarelli/headbreaker/discussions/55#discussioncomment-5423295.

**Comment by flbulgarelli** [comment]
:wave: Hi! Please see https://github.com/flbulgarelli/headbreaker/discussions/55#discussioncomment-5423295.

Hope it helps!
